ods_ci/tests/Resources/Files/accelerator/qe_accelerator_profile_2.yaml

```
@@ -0,0 +1,17 @@
+# File to provision an Accelerator Profile instance in test:
+#   "Modify An Accelerator Profile Using "Accelerator Profiles" Administration UI"
+apiVersion: dashboard.opendatahub.io/v1
+kind: AcceleratorProfile
+metadata:
+  name: qeacceleratorprofile2
```

Just an example, but relevant for all yamls:
```suggestion
  name: accelerator-profile-2
```

---

I do not think It's a good idea to delete the "qe". The idea here is to have some kind of string which is unique for this feature. Something like  "qe-accelerator-profile-XXX" so when this feature (or test suite) has ended, we can filter all which start with "qe-accelerator-profile-" and delete it (as we are already doing). If we delete the "qe" and run this is some productuion cluster and there are AP instances which start with name "accelerator-profile" we will delete them